app/assets/javascripts/components/uploads/upload_viewer.jsx

```diff
@@ -41,16 +78,26 @@ const UploadViewer = createReactClass({
     const source = _.get(metadata, 'imageinfo[0].extmetadata.Credit.value');
     const license = _.get(metadata, 'imageinfo[0].extmetadata.LicenseShortName.value');
 
+    const eyeIcon = <span><img className="eye-icon" src="/assets/images/icon-eye.png" alt="View icon" /></span>;
```

Comment:
Icons should be SVG, so that they'll scale well on lots of different screen sizes.